src/HtmlReader/lib.ts

```diff
@@ -0,0 +1,347 @@
+import { Injectable } from '../Readium/Injectable';
+import { Locator } from '../Readium/Locator';
+import { WebpubManifest, ColorMode, FontFamily } from '../types';
+import { ReadiumLink } from '../WebpubManifestTypes/ReadiumLink';
+import { ActiveState } from './types';
+
+/**
+ * Constants
+ */
+export const FONT_SIZE_STEP = 4;
+
+/**
+ * If we provide injectables that are not found, the app won't load at all.
+ * Therefore we will not provide any default injectables.
+ * @todo - this is not true in our custom renderer. We should provide default injectables.
+ */
+export const defaultInjectables: Injectable[] = [];
```

Comment:
what is the difference between fixed injectibles and non-fixed injectibles?

Reply:
One is for fixed layout and the other is for reflowable books